### ISSUE TITLE
Clippy warns of unused variable in the code generated by #[filter]

### ISSUE THREAD

**Issue by brunocodutra** [frozenlib/test-strategy#6]
As of the latest nightly Clippy is throwing warns like 

```
warning: unused variable: `args`
  --> lib/transposition/iter.rs:45:19
   |
45 |         #[filter(#tt.capacity() > 1)]
   |                   ^^^^^^^^^^^^^^^^^ help: if this is intentional, prefix it with an underscore: `_args`
```

I suppose it stems from either [here](https://github.com/frozenlib/test-strategy/blob/8fa088aba61b9cc57d9d90f92a68887e550472f0/src/arbitrary.rs#L286) or [here](https://github.com/frozenlib/test-strategy/blob/8fa088aba61b9cc57d9d90f92a68887e550472f0/src/arbitrary.rs#L272), but I'm not too familiar with the quote crate and may be reading this wrong.

**Comment by frozenlib** [comment]
Thanks for the report.

I have added an `#[allow(unused)]` there with d1a75d4a08b78477b4b4e88df782986015fa9576, as that is indeed the cause of the problem.

**Comment by brunocodutra** [comment]
Thanks for the quick turnaround!

Are you planning a point fix release anytime soon?

**Comment by frozenlib** [comment]
I have now released v0.2.1, which includes this fix.
